csharp/src/Ice/Connection.cs

```diff
@@ -157,6 +157,8 @@ internal bool Active
         private Task _receiveTask = Task.CompletedTask;
         private readonly Dictionary<long, (TaskCompletionSource<IncomingResponseFrame>, bool)> _requests =
             new Dictionary<long, (TaskCompletionSource<IncomingResponseFrame>, bool)>();
+        private readonly Dictionary<long, CancellationTokenSource> _dispatch =
```

Comment:
We should sort these fields in alphabetical order. This new _dispatch field represents requests being dispatched and is presumably protected by _mutex? What does _requests represent?

Reply:
_requests represent the client-side pending outgoing requests and we keep track of the task completion source for these to be able to complete them when the response is received.  Both _requests and _dispatch will eventually  hold additional information necessary for supporting stream parameters and data frames.